bind/gencffi_cdef.go

```diff
@@ -99,6 +198,8 @@ func (g *cffiGen) genCdefMethod(f Func) {
 	g.wrapper.Printf("extern %[1]s cgo_func_%[2]s(%[3]s);\n", cdef_ret, f.id, paramString)
 }
 
+// genCdefStructMemberGetter generates
+// C definitions of Getter/Setter for Go struct.
```

Comment:
`s/for Go struct/for a Go struct/`
also, please everything on one line (ie: no new line after `generates`.)